pantheon-bundle/src/main/resources/SLING-INF/content/docs/assemblies/assembly-pantheon-help.adoc

```diff
@@ -1,33 +1,93 @@
-ifdef::context[:parent-context: {context}]
+// This assembly is included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+:toc: left
+:numbered:
 
 include::../attributes.adoc[]
 
-[id="help"]
-= Help
+ifdef::context[:parent-context: {context}]
+
+[id="assembly-help_{context}"]
+= Pantheon Help Page
 
 :context: assembly-help
 
 [role="_abstract"]
 Welcome to {ProductShortName} help.
 
+This document describes the main concepts, requirements, and actions in {ProductShortName}.
+
 include::modules/ref_prerequisites.adoc[leveloffset=+1]
 
-// Link to Overview of Pantheon
+== Overview of {ProductShortName}
+
+This overview describes {ProductShortName} and the key terms associated with it.
+ 
+include::modules/con_pantheon.adoc[leveloffset=+2]
+
+include::modules/ref_pantheon-user-interface.adoc[leveloffset=+2]
+
+include::modules/con_about_products.adoc[leveloffset=+2]
+
+include::modules/con_user-roles.adoc[leveloffset=+2]
+
+== Overview of modules
+
+This overview describes what a module is in the context of {ProductShortName} and how {ProductShortName} tells modules apart.
+
+include::modules/con_definition-of-a-module.adoc[leveloffset=+2]
+
+include::modules/con_definition-of-metadata.adoc[leveloffset=+2]
+
+include::modules/ref_identifying-module-types.adoc[leveloffset=+2]
+
+== Configuring your repository
+
+Before you can add {ContentTerm} to {ProductName}, you must configure your documentation repository as a content source for {ProductShortName} and then create a product and product version in {ProductShortName} to associate with your {ContentTerm}.
+
+* xref:configuring-a-repository-as-a-content-source-for-pantheon_assembly-help[Configuring your documentation repository as a content source for {ProductShortName}.]
+* xref:creating-a-product_assembly-help[Creating a product] in {ProductShortName} for your {ContentTerm}
+* xref:versioning-a-product_assembly-help[Creating a product version] in {ProductShortName} for your {ContentTerm}
+
+include::modules/con_pantheon_yaml_file.adoc[leveloffset=+2]
+
+include::modules/proc_config_repo.adoc[leveloffset=+2]
+
+include::modules/proc_creating-a-product.adoc[leveloffset=+2]
+
+include::modules/proc_versioning_a_product.adoc[leveloffset=+2]
+
+include::modules/proc_importing-content.adoc[leveloffset=+1]
+
+[[searching-for-content_{context}]]
+
+== Searching for {ContentTerm}
+
+The *{ContentTerm}* tab allows you to search for content in {ProductShortName} by simple keyword or free text search, and by applying a number of different filters to search and to sort the results.
```

Comment:
I think this would render to "The content tab" - right? It might be more accurate to say "the search tab," but "content" is the name of the grouping so perhaps your wording was intentional here.

Reply:
I believe "the search tab" is more accurate here. Updating.